README.md

```diff
@@ -78,145 +135,201 @@ See [third_party_integration](./third_party_integration) for examples of code mo
 ## System requirements
 - Ubuntu\* 18.04 or later (64-bit)
 - Python\* 3.6.2 or later
-- NVidia CUDA\* Toolkit 10.2 or later^
-- PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)
+- Supported frameworks:
```

Comment:
I think it should be rephrased as there is no support for versions >1.8.1

Reply:
What do you think about the following formulation `PyTorch\* >=1.5.0, <=1.8.1 (1.8.0 not supported)`?

Reply:
It's ok with me